tests/functional/core.js

```diff
@@ -1,384 +1,139 @@
-
-       this.sendKeys('body', casper.page.event.key.Left);
-       this.sendKeys('body', casper.page.event.key.Left);
-
-   }).then(function() {
-
-       test.assertExists('[id="1"].active:not(.visited)', 'Slide #1 is active but not visited');
-       test.assertExists('#MyID.visited:not(.active)', 'Slide #MyID is visited but not active');
-       test.assertExists('[id="3"].visited:not(.active)', 'Slide #MyID is visited but not active');
-
-   }).run(function() { test.done() }).clear();
-});
-
-casper.test.begin(
-// ------------------------------------------------------------------
-    'State classes while moving forward and backward in List mode', 3,
-// ------------------------------------------------------------------
-    function suite(test) {
-    casper.start('tests/functional/core-list.html#1').then(function() {
-
-        this.sendKeys('body', casper.page.event.key.Right);
-        this.sendKeys('body', casper.page.event.key.Right);
-        this.sendKeys('body', casper.page.event.key.Left);
-        this.sendKeys('body', casper.page.event.key.Left);
-
-    }).then(function() {
-
-        test.assertExists('[id="1"].visited.active', 'Slide #1 is visited and active');
-        test.assertExists('#MyID.visited:not(.active)', 'Slide #MyID is visited but not active');
-        test.assertExists('[id="3"].visited:not(.active)', 'Slide #MyID is visited but not active');
-
-    }).run(function() { test.done() }).clear();
-});
-
-casper.test.begin(
-// ------------------------------------------------------------------
-    'State classes while moving forward and backward in Full mode', 3,
-// ------------------------------------------------------------------
-    function suite(test) {
-    casper.start('tests/functional/core-list.html?full#1').then(function() {
-
-        this.sendKeys('body', casper.page.event.key.Right);
-        this.sendKeys('body', casper.page.event.key.Right);
-        this.sendKeys('body', casper.page.event.key.Left);
-        this.sendKeys('body', casper.page.event.key.Left);
-
-    }).then(function() {
-
-        test.assertExists('[id="1"].visited.active', 'Slide #1 is visited and active');
-        test.assertExists('#MyID.visited:not(.active)', 'Slide #MyID is visited but not active');
-        test.assertExists('[id="3"].visited:not(.active)', 'Slide #MyID is visited but not active');
-
-    }).run(function() { test.done() }).clear();
+describe('core', function() {
+    beforeEach(function() {
+        //force page reload
+        browser.url('about:blank');
+    });
+
+    it('List mode auto-initialisation in lack of mode class', function() {
+        browser.url('/core.html');
+        browser.waitForVisible('.shower.list');
+    });
+
+    // test seems to be outdated
+    xit('Full mode initialisation by mode class name', function() {
+        browser.url('/core-full.html');
+        browser.waitForVisible('.shower.full');
+        browser.getUrl().should.match(/\?full/, 'URL in Full mode')
+    });
```

Comment:
I guess that url `my-slides.html?full` doesn't trigger full mode anymore. Should I remove tests on it?